### PR TITLE
Update deploy to Elroy when err envapi

### PR DESCRIPTION
**Pull request by manute**
https://github.com/InVisionApp/kit-server/issues/284

### CODE REVIEW

File: src/lib/deploymentizer.js [hunk 1]
```diff
@@ -110,6 +110,19 @@ class Deploymentizer {
       if (this.options.configPlugin) {
         let pluginOpts = this.options.configPlugin.options;
         pluginOpts.events = this.events;
+        if (
+          this.options.elroyUrl &&
+          this.options.elroySecret &&
+          this.options.elroyOnly
```
Comment (erutherford) [inline src/lib/deploymentizer.js]: I thought `this.options.elroyOnly` was only used for syncs?  Is it also used during normal deploys?

Comment (manute) [reply src/lib/deploymentizer.js]: oh man I don't know.. 🤦‍♂️

Comment (erutherford) [reply src/lib/deploymentizer.js]: It's not in the docs :/

Comment (erutherford) [reply src/lib/deploymentizer.js]: [https://github.com/InVisionApp/kit-deploymentizer/blob/c5ca99485a0ac99dc1e75c4d0e5670c457bd617c/src/deploymentizer#L107-L112](this) seems to imply that it's only for an Elroy sync

**Comment by manute** [comment]
closing this in favor of https://github.com/InVisionApp/kit-server/pull/307

**Pull request closed without merging**